sigs.yaml

```diff
@@ -3995,7 +4008,7 @@ committees:
     - github: aojea
       name: Antonio Ojea
       company: Google
-      email: antonio.ojea.garcia@gmail.com
+      email: aojea@google.com
```

Comment:
@aojea this was the one set the most for you, and the oldest one, but we can update it to either (just need it to be the same for all so we don't duplicate to CNCF)

The rest of these were only "inconsistent" for having empty values.

Reply:
I'm fine with any